bitsail-connectors/connector-redis/src/main/java/com/bytedance/bitsail/connector/redis/config/RedisOptions.java

```diff
@@ -58,7 +58,7 @@ public class RedisOptions implements Serializable {
   private RowTypeInfo rowTypeInfo;
 
   /**
-   * Log interval of pipelines.
+   * Log interval of pipelines, Send log every 'logSampleInterval' times flush (Each flush corresponds to one pipeline)
```

Comment:
Log sample every N batch.

Reply:
I modified it to 
![image](https://user-images.githubusercontent.com/47937942/200501162-c94a2402-e50e-438c-848b-9d202fde37f9.png)